Resources/Prototypes/_RMC14/Roles/Jobs/Other/Merc/mercenary_engineer.yml

```
@@ -0,0 +1,76 @@
+- type: job
+  parent: CMJobBase
+  id: RMCEliteMercMEngineer
```

Is the extra M intended? The other ids in this file don't have it
```suggestion
  id: RMCEliteMercEngineer
```